### PR TITLE
Validate the 'cluster.remote_info' API

### PR DESCRIPTION
**Pull request by sethmlarson**
As titled.

### CODE REVIEW

**Review by philkra — APPROVED** [review]
LGTM

**Review by philkra — APPROVED** [review]
LGTM

**Comment by delvedor** [comment]
> Where should this response go to make sure we're actually testing proper data?

You can create a custom test in the flight recorder.

File: specification/cluster/remote_info/ClusterRemoteInfoRequest.ts [hunk 1]
```diff
@@ -18,14 +18,16 @@
...
  * @rest_spec_name cluster.remote_info
  * @since 6.1.0
- * @stability TODO
+ * @stability stable
  */
 export interface Request extends RequestBase {
-  body?: {
-    stub: string
-  }
+  body: Void
```
Comment (delvedor) [inline specification/cluster/remote_info/ClusterRemoteInfoRequest.ts:32]: There is no need to use `body: Void`, if the API has no body at all, all you need to do is to not define the body :)
```suggestion
```

**Review by delvedor — CHANGES_REQUESTED** [review]

File: specification/cluster/remote_info/ClusterRemoteInfoRequest.ts (hunk 1 above)
Comment (sethmlarson) [reply specification/cluster/remote_info/ClusterRemoteInfoRequest.ts:32]: Sure, either way I get a validation error unfortunately.

Comment (delvedor) [reply specification/cluster/remote_info/ClusterRemoteInfoRequest.ts]: Related: https://github.com/elastic/elasticsearch-specification/issues/580

**Review by delvedor — APPROVED** [review]
LGTM

**Pull request merged**